k9mail/src/androidTest/java/com/fsck/k9/crypto/OpenPgpUtilsTest.java

```diff
@@ -0,0 +1,53 @@
+package com.fsck.k9.crypto;
+
+
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openintents.openpgp.util.OpenPgpUtils;
+
+import static junit.framework.Assert.assertEquals;
+
+
+@RunWith(AndroidJUnit4.class)
+public class OpenPgpUtilsTest {
+
+    @Test
+    public void splitCompleteUserIdShouldReturnAll3Components() throws Exception {
+        OpenPgpUtils.UserInfo info = OpenPgpUtils.splitUserId("Max Mustermann (this is a comment) <max@example.com>");
+
+        assertEquals(info.name, "Max Mustermann");
```

Comment:
the `assertEquals` param order is `expected, actual`, so this (and all other assertions) should be written as
```java
assertEquals("Max Mustermann", info.name);
```